samples/Nancy.Demo.Hosting.Kestrel/HomeModule.cs

```diff
@@ -8,7 +9,19 @@ public HomeModule()
         {
             Get("/", (args, ct) => Task.FromResult("Hello from Nancy running on CoreCLR"));
 
-            Get("/conneg/{name}", (args, token) => Task.FromResult(new Person() { Name = args.name }));
+            Get("/conneg/{name}", (args, token) => Task.FromResult(new Person() { Name = args.name });

```

Comment:
That's not in scope for this PR. That example was from the old
async routing which can be changed in a new PR

On Sunday, 29 May 2016, Andreas Håkansson <notifications@github.com> wrote:

> In samples/Nancy.Demo.Hosting.Kestrel/HomeModule.cs
> <https://github.com/NancyFx/Nancy/pull/2469#discussion_r65005201>:
>
> > @@ -8,7 +9,19 @@ public HomeModule()
> >          {
> >              Get("/", (args, ct) => Task.FromResult("Hello from Nancy running on CoreCLR"));
> >
> > -            Get("/conneg/{name}", (args, token) => Task.FromResult(new Person() { Name = args.name }));
> > +            Get("/conneg/{name}", (args, token) => Task.FromResult(new Person() { Name = args.name });
>
> Same as below, no need for the Task-based overload
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/NancyFx/Nancy/pull/2469/files/6f74b42440552b867bfcf94708c27e63874df264#r65005201>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAGapvGj6CLZKNCtT3pDN0UE951co5uYks5qGZfugaJpZM4Io0sz>
> .
>